pandas/core/indexes/base.py

```diff
@@ -2289,26 +2334,46 @@ def union(self, other, sort=None):
         >>> idx2 = pd.Index([3, 4, 5, 6])
         >>> idx1.union(idx2)
         Int64Index([1, 2, 3, 4, 5, 6], dtype='int64')
+
+        >>> idx1 = pd.Index(['a', 'b', 'c', 'd'])
```

Comment:
a 1-line comment the various examples maybe useful

Reply:
done